chapters/06-summary.Rmd

```diff
@@ -21,5 +22,6 @@ Supported algorithms                   Exact          Tree    Exact, FFT  Exact,
 
 Table: (\#tab:finalComparison) Comparison between KDE implementations (NR: normal reference rules, namely Scott/Silverman, CV: Cross Validation, ISJ: Improved Sheater Jones according to Botev et al.)
 
-The newly proposed KDE implementation (based on TensorFlow and zfit) achieves state-of-the-art accuracy as well as efficiency for large one-dimensional data ($n \geq 10^8$). Furthermore it is designed to be run on parallel on multiple machines/GPUs. It is therefore optimally suited for very large datasets, as the ones produced in experimental high energy physics.
+In conclusion, the new newly proposed kernel density estimation implementation based on TensorFlow and zfit achieved state-of-the-art accuracy as well as efficiency for large one-dimensional data ($n \geq 10^8$). In addition, the fast evaluation phase of the proposed methods is useful for log-likelihood or $\chi^2$-squared fits. Furthermore, the proposed improved Sheather-Jones algorithm implementation shows state-of-the-art accuracy in general while imposing only a minor runtime cost. Since all methods are based on TensorFlow and zfit, they are optimally suited for parallel processing on multiple CPUs and GPUs. Therefore this new implementation is optimally suited for kernel density estimation in scientific fields that deal with large datasets, as high energy physics for example.
```

Comment:
Yes, but the second part of the paragraph seems unsatisfying. Next to the abstract, this is maybe the most important paragraph. It's the end of the thesis. Maybe polish this 2-3 sentences.

P.S. surely remove the "for example". It makes it a very weak statement, you can afford a stronger one